src/main/java/entity/Equipment.java

```diff
@@ -0,0 +1,52 @@
+package entity;
+
+public class Equipment extends Item implements Stats {
+    private final String statType; // the type of stat
+    private int statValue; // the value of the stat
+
+    /**
+     * Creates a new entity.Equipment object.
+     * @param name the name of the entity.Equipment.
+     * @param num the number of equipments.
+     * @param statType the type of stat associated with the entity.Equipment.
+     * @param statValue the value of the stat associated with the entity.Equipment.
+     */
+    public Equipment(String name, int num, String statType, int statValue){
```

Comment:
if equipment class is abstract, we wont need this constructor

Reply:
I agree that Equipment should be abstract, but I would prefer to keep the constructor because of statType and statValue. Without the constructor, every subclass of Equipment would have to implement its own constructor in order to have their statType and statValue attributes.